conan/tools/microsoft/msbuilddeps.py

```diff
@@ -25,7 +25,6 @@ class MSBuildDeps(object):
           <PropertyGroup Label="ConanVariables">
             <Conan{{name}}RootFolder>{{root_folder}}</Conan{{name}}RootFolder>
             <Conan{{name}}BinaryDirectories>{{bin_dirs}}</Conan{{name}}BinaryDirectories>
-            <Conan{{name}}Dependencies>{{dependencies}}</Conan{{name}}Dependencies>
```

Comment:
This is being removed, as it is not used at all by Conan, and it was flawed, as it contained a mix of xml-replaced names, components names, etc. This information is already encoded in the ``<Imports>`` sections of the .props files explicitly.